cli/tauri-cli/src/bundle/osx_bundle.rs

```diff
@@ -201,7 +201,7 @@ fn copy_frameworks_to_bundle(bundle_directory: &Path, settings: &Settings) -> cr
   for framework in frameworks.iter() {
     if framework.ends_with(".framework") {
       let src_path = PathBuf::from(framework);
-      let src_name = src_path.file_name().unwrap();
+      let src_name = src_path.file_name().expect("Couldn't get file name");
```

Comment:
```
Could not get filename
```
Like in L248 in this file

Reply:
alright, differentiated them with icon and framework respectively.